project/AppDependencies.scala

```diff
@@ -7,9 +7,9 @@ object AppDependencies {
   private val testScope = "test, it, component"
 
   val compile = Seq(
-    "uk.gov.hmrc"                    %%  "simple-reactivemongo"  % "7.30.0-play-26",
+    "uk.gov.hmrc"                    %%  "simple-reactivemongo"  % "7.30.0-play-27",
```

Comment:
Here I have a problem. Sbt for me is trying to download this library with scala version 2.11, which not exists, because play 2.7 require scala 2.12